Resources/Prototypes/_NF/Roles/Jobs/Pirates/pirate_captain.yml

```
@@ -5,7 +5,7 @@
   playTimeTracker: JobPirateCaptain
   requirements:
     - !type:OverallPlaytimeRequirement
-      time: 32400 # 9 hours - mono
+      time: 86400 # 24 hours - mono
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)